addons/binding/org.openhab.binding.camera/ESH-INF/config/cameraConfig.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+    <config-description uri="thing-type:camera:camera">
+        <parameter name="poll" unit="s" type="integer">
+            <label>Poll time:</label>
+            <description>Interval of image being refreshed in seconds</description>
+            <default>5</default>
+        </parameter>
+        <parameter name="urlSnapshot" type="text">
+            <label>Snapshot Url:</label>
+            <description>Url for receiving the camera snapshot</description>
+            <default>http://ip/image.jpg</default>
+        </parameter>
+        <parameter name="urlUsername" type="text">
+            <label>Username:</label>
+            <description>Username for accessing camera (empty for standard)</description>
+            <default></default>
+        </parameter>
+        <parameter name="urlPassword" type="text">
+            <label>Password:</label>
+            <description>Password for accessing camera (empty for standard)</description>
+            <default></default>
+        </parameter>
+        <parameter name="urlVideo" type="text">
+            <label>Video Url:</label>
+            <description>Url for receiving the camera video. ..TODO..</description>
```

Comment:
There is a TODO, what should be done and can it be completed before merging the PR?

Reply:
This is on top of my list but really can not make it into release, I commented out in the meanwhile.